src/content/docs/cypher/difference.md

```diff
@@ -0,0 +1,83 @@
+---
+title: Difference in Cypher Implemetations
+---
+
+# Difference from Neo4j's Cypher Implementations
+
+## Schema 
+
+Different from Neo4j, Kuzu requires a schema to be defined before any data can be inserted into the database. The schema provides a logical grouping of node and relationship tables, along with their associated properties and data types that define the structure of the graph database. See [DDL](/cypher/data-definition/create-table.md) for more information. 
+
+
+## Clauses
+
+### CREATE and MERGE
+
+It is recommended to always specifying node and relationship labels explicitly in the `CREATE` and `MERGE` clause. If not specified, Kuzu will try to infer the label by looking at the schema.
```

Comment:
> Kuzu will try to infer the label by looking at the schema

Do we support this? How do we infer from the schema?

Reply:
As a simple example, if the database only contains 1 node table and 1 rel table, the following will succeed.

```
CREATE ()-[]->()
```